### PR TITLE
Remove `hashicorp/template` unnecessary dependency

### PR DESCRIPTION
**Pull request by frqnck**

### CODE REVIEW

**Review by nitrocode — APPROVED** [review]

**Comment by akshaychopra5207** [comment]
Can Anybody merge this PR? template provider is breaking for arm MAC M1 and also it is deprecated

**Comment by nitrocode** [comment]
@akshaychopra5207 we cannot merge this yet since the releases are in pre-release due to the old sg module. We need to first upgrade the sg module to latest version, then we can allow this change to go through.

**Comment by akshaychopra5207** [comment]
any  expected date for that ? @nitrocode

**Comment by akshaychopra5207** [comment]
otherwise I have to create my own module as a workaround

**Comment by nitrocode** [comment]
You're free to fork the code to your own module. We also accept pull requests.

See https://github.com/cloudposse/terraform-aws-efs/pull/94 for references to other PRs which these can be used as a guide.

TLDR to migrate out of pre-release:

* Deprecate old variables from before pre-release
* Merge any outstanding tested PRs
* Use best practices
* Create migration guide
* Ping us to review

cc: @Nuru 

Or wait for one of the cloudposse devs to migrate this. It's on our list.

**Review by nitrocode — CHANGES_REQUESTED** [review]
aa

**Comment by flowinh2o** [comment]
Can this be merged yet?

**Comment by Gowiem** [comment]
@flowinh2o unfortunately, this module is currently frozen as we're trying to upgrade all modules that use our security-group module to v4 and it is a time consuming process. Because our module library is so large, this takes time. We'll work on getting to this module at some point soon and will get this merged ASAP. Follow this PR for future updates.

**Comment by jabcoder** [comment]
Is there a workaround for this while we wait?

**Comment by austin-engle** [comment]
Looks like we are out of pre-release and this might be able to be merged in? Also can we ensure that if template is removed it also gets removed from the example

**Comment by nitrocode** [comment]
I believe we're still waiting on PR #41 before this PR can be mergee. Cc: @milldr and @Nuru
